src/rospkg/os_detect.py

```diff
@@ -695,6 +697,7 @@ def get_codename(self):
 OS_SLACKWARE = 'slackware'
 OS_UBUNTU = 'ubuntu'
 OS_WINDOWS = 'windows'
+OS_CLEARLINUX = 'Clear Linux OS for Intel Architecture'
```

Comment:
This would be better as a single tag like string not a full sentence with spaces etc.